tests/epyccel/test_epyccel_variable_annotations.py

```diff
@@ -10,6 +10,14 @@
 from pyccel.errors.errors import PyccelSemanticError, Errors
 from pyccel.decorators import allow_negative_index, stack_array
 
+def parametrize_languages():
+    return pytest.mark.parametrize('language', [
+        pytest.param("c", marks=pytest.mark.c),
+        pytest.param("fortran", marks=[
+            pytest.mark.skip(reason="Variable declaration not implemented in fortran"),
+            pytest.mark.fortran]),
+        pytest.param("python", marks=pytest.mark.python)
+    ])
```

Comment:
An alternative to this (and a more standard way of handling it) would be to create a [`fixture`](https://docs.pytest.org/en/stable/explanation/fixtures.html) with a different name that is used until Fortran can be included.

Reply:
This is what you have done for functionals

Reply:
what do you think about this:
721dfd8f5fde86dfb3602051f83d9156da89cce9

Reply:
I'm not sure why you created a new file? Presumably this is because you kept the same name for the fixture? It is hard enough to locate tests at the moment I would rather not make this work. I think it would be better to rename the fixture. E.g. `stc_language`
This can still easily be changed when it's fixed using replace all

Reply:
tell me what you think about this version:
cd22d9552fa7c174faa97aaba068fe8ff2acd247

Reply:
Looks good